testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/OAuth2DeviceAuthorizationGrantTest.java

```diff
@@ -75,6 +75,7 @@ public class OAuth2DeviceAuthorizationGrantTest extends AbstractKeycloakTest {
     public static final String DEVICE_APP = "test-device";
     public static final String DEVICE_APP_PUBLIC = "test-device-public";
     public static final String DEVICE_APP_PUBLIC_CUSTOM_CONSENT = "test-device-public-custom-consent";
+    private static final String shortDeficeFlowUrl = "https://keycloak.org/device";
```

Comment:
```suggestion
    public static final String SHORT_DEVICE_FLOW_URL = "https://keycloak.org/device";
```

Reply:
Why make public a variable that is only used in this class?

Reply:
Yes, you're right. I was just wondering about making it consistent with other constants.

So when you're on this, could you leave it private, but possibly also make other variables in this class private (DEVICE_APP and others) as they are also used just in this test?

Will be good to fix typo (defice) and use the notation for constants like SHORT_DEVICE_FLOW_URL.